modules/cavalier/hm.nix

```
@@ -0,0 +1,24 @@
+{ config, lib, ... }:
+
+{
+  options.stylix.targets.cavalier = {
+    enable = config.lib.stylix.mkEnableTarget "Cavalier" true;
+  };
+
+  config =
+    let
+      cfg = config.stylix.targets.cavalier;
+    in
+    lib.mkIf (config.stylix.enable && cfg.enable) {
+      programs.cavalier.settings.general = {
+        ColorProfiles =
+          with config.lib.stylix.colors;
+          lib.singleton {
+            Name = "Stylix";
+            FgColors = lib.singleton base05;
+            BgColors = lib.singleton base00;
```

This here should be much simpler:

```suggestion
            FgColors = [base05];
            BgColors = [base00];
```

---

single values should be `[x]`, attrsets should be `lib.singleton { foo = bar; }`

---

Oh, actually didn't know that. Thanks for clarifying!